packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -66,97 +68,141 @@ export class ListboxOption extends FoundationElement {
     @attr({ mode: "boolean" })
     public disabled: boolean;
     protected disabledChanged(prev, next): void {
-        if (this.proxy instanceof HTMLOptionElement) {
+        if (this.hasProxy) {
             this.proxy.disabled = this.disabled;
         }
     }
 
+    public get form(): HTMLFormElement | null {
+        return this.proxy?.form;
+    }
+
+    private get hasProxy() {
+        return this.proxy instanceof HTMLOptionElement;
+    }
```

Comment:
Checking for the proxy before accessing it always kind of bugged me. I also wondered, what if there were other reasons to not update the proxy, we'd have to change it in multiple places. This fixes that problem, but then shouldProxy() would be a more descriptive name.

It still leaves us with a bunch of if(this.hasProxy()) {} wrappers in code. Possibly a this.setProxyAttribute(attribute, value); method that we could blindly call? And then we could put what ever logic we want in the setProxyAttribute) method including this.proxy istanceof HTMLOptionElement. Just food for thought.

This needs a boolean return type.